src/create-verify-archive.js

```diff
@@ -76,6 +77,49 @@ exports.createCommand = function*(argv) {
     var absOutDir = path.resolve(outDir);
 
     yield repoutil.forEachRepo(repos, function*(repo) {
+        if(isWin) {
```

Comment:
Thanks for the clarification @daserge 
On Mac by default using git from XCode Tools, both of the settings core.autocrlf and core.eol are not set at all, so I would think let's leave this check only for Win as you have it here, go ahead and merge this PR